activerecord/lib/active_record/migration.rb

```diff
@@ -885,13 +885,14 @@ def method_missing(method, *arguments, &block)
 
     def copy(destination, sources, options = {})
       copied = []
+      schema_migration = options[:schema_migration]
```

Comment:
I don't see the method call for this method passing the `schema_migration` on this PR. Should not we change it? Or it is just relying on the fact that the methods being called on `MigrationContext` don't use the `schema_migration` information? Maybe we should just default to `SchemaMigration` on this method like we did on `current_version` so at least things will not explode if `migrations` start to use the `schema_migration ` argument. There is also a test setting `schema_migration` to `nil` that we probably should use `SchemaMigration`.

Reply:
I purposefully didn't change the method signature because `copy` is a public method. Updated this to default to not nil.